Library/Formula/speedread.rb

```diff
@@ -0,0 +1,19 @@
+class Speedread < Formula
+  desc "Simple terminal-based rapid serial visual presentation (RSVP) reader"
+  homepage "https://github.com/pasky/speedread"
+  url "https://github.com/pasky/speedread/archive/v1.0.tar.gz"
+  version "1.0"
```

Comment:
@hashier It won’t work if you’re not a maintainer :wink: 
The build has been triggered but there are some PRs before it in the queue so we have to wait.